### PR TITLE
Adding Maintainers Guide part of Docs folder

### PR DESCRIPTION
**Pull request by kannankvs**
Signed-off-by: kannankvs <kannan_kvs@dell.com>

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by kannankvs** [comment]
Was closed by mistake. Re opening it again

**Review by kerry-meyer — COMMENTED** [review]
Please respond to each of the questions below unless the answer to the first one is "Yes".

Should this PR be "Closed"? (It can be re-opened later with actual content, but since it clearly isn't being posted for review, it would help with our tracking to either close it for now or provide real content.)

Is there a distinction being made here between a "Maintainers Guide" and a "Contributors Guide"? (Note: There is no expectation of having external "maintainers". If a "maintainers'" guide is viewed as something we should have, then any guidelines defined now would be relevant regardless of whether or not there are external contributors.)

Does the comment posted in this file mean that PR 134 (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/134  : Added Contributors Guide to docs folder) should be deferred or closed for now?

File: docs/MAINTAINER_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+At this point, only dell team is contributing. A guide shall be provided when it is opened for external maintainers.
```
Comment (stalabi1) [inline docs/MAINTAINER_GUIDE.md:1]:
```suggestion
At this point, only the Dell team is contributing. A guide shall be provided when it is opened for external maintainers.
```

**Comment by kannankvs** [comment]
We are unable to commit in this PR due to "head repository deletion" issue. Hence we raised a new [PR#164](https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/164) and copied all the comments from this PR to the new PR. This PR is no more valid and hence we are closing it.

**Pull request closed without merging**